GWAS/PCA.ipynb

```diff
@@ -36,7 +36,7 @@
     "## Input\n",
     "\n",
     "1. Genotype in PLINK format: could be different from bfile in the case of using for e.g. exome data / imputed genotype data\n",
-    "2. Phentype files with population information and possibly disease or other labelling information, must contain columns named 'FID' and 'IID' for family ID and individual ID. **FIXME: can we require only IID and in our code create FID by setting it to IID if FID does not exist?**\n",
```

Comment:
@haoyueshuai i put a FIXME there for a reason. I raised a question if we can remove the need to have FID and only input IID. Please bring the FIXME back until the question i raised was addressed to